sdk/cosmos/.azure-pipelines/cosmos.test.yml

```diff
@@ -3,7 +3,7 @@ trigger: none
 pr:
   branches:
     include:
-    - master
+    - cosmos_v3
```

Comment:
Should this be changed?

Reply:
Yes, but not till right before/after the merge to master I think?